### DISCUSSION TITLE
Can Jellyfin send audio via bluetooth on a RPi4 Docker install?

### DISCUSSION THREAD

**Discussion by mikae1** [jellyfin/jellyfin#9157] [Q&A] [answered]
Hi!
If I install Jellyfin as a Docker image on my RPi4, could I send the audio from the RPi via the built-in Bluetooth to a Logitech Bluetooth receiver connected to my stereo amplifier a few meters away?

**Comment by kuduacz** [comment]
Wish to know it too

**Comment by cvium** [comment, marked as answer]
There is no bluetooth connectivity in Jellyfin.
